api/client/src/main/java/org/projectnessie/client/http/impl/jdk11/JavaResponseContext.java

```diff
@@ -75,6 +84,21 @@ public URI getRequestedUri() {
     return response.uri();
   }
 
+  public void close(Exception error) {
+    if (error != null) {
+      try {
+        LOGGER.debug(
+            "Closing unprocessed input stream for {} request to {} delegating to {} ...",
```

Comment:
How do we know the input stream is "unprocessed"?

Reply:
If an error has been thrown, it is absolutely certain that the response body won't be read because the caller has no access to it.